### DISCUSSION TITLE
How to re-size an image in a post?

### DISCUSSION THREAD

**Discussion by kaon1** [CaiJimmy/hugo-theme-stack#721] [General]
When I embed an image with `![]()` it gets sized down. 

I would like to keep original or re-size it. What is the syntax for this?

**Comment by CaiJimmy** [comment]
I'm afraid there's no Markdown syntax for that as I am aware.

**Comment by TigGrak** [comment]
md can not do this but probably set ```markup.goldmark.renderer.unsafe = true``` in your config file and use html

```html
<img src="your pic" width="50%" height="50%"/>
```

can do it
